client-templates/nexus3/.env.sample

```diff
@@ -20,10 +20,17 @@ BROKER_HEALTHCHECK_PATH=/healthcheck
 
 # Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials and http protocol
 # This URL substitution is required for NPM integration
-# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
+# RES_BODY_URL_SUB=https://<your.nexus.hostname>/repository
 
 # Nexus validation url, checked by broker client systemcheck endpoint
 # readonly users must have 'nx-metrics-all' privilege enabled to access
 # https://support.sonatype.com/hc/en-us/articles/226254487-System-Status-and-Metrics-REST-API
+
+# NOTE! If NOT using username and password in BASE_NEXUS_URL, because your Nexus server is open on a private network,
+# then use '/rest/v1/status' for BROKER_CLIENT_VALIDATION_URL.
```

Comment:
use `'/service/rest/v1/status'` for better readability, perhaps? (according to the REST API docs)